test/functional/mining_prioritisetransaction.py

```diff
@@ -131,7 +134,10 @@ def run_test(self):
         self.relayfee = self.nodes[0].getnetworkinfo()['relayfee']
 
         utxo_count = 90
-        utxos = create_confirmed_utxos(self, self.relayfee, self.nodes[0], utxo_count)
+        utxos = self.wallet.send_self_transfer_multi(from_node=self.nodes[0], num_outputs=utxo_count)['new_utxos']
+        self.generate(self.wallet, 1)
+        assert_equal(len(self.nodes[0].getrawmempool()), 0)
```

Comment:
I really wonder if it is worth it to move this to `create_confirmed_utxos`? Seems 2-3 LOC duplicated everywhere are fine compared to 1 LOC + import + definition in separate file?

Maybe just remove `create_confirmed_utxos`?

Reply:
Right, I agree that it's not worth it, calling `send_self_transfer_multi` with a subsequent `generate` is simple enough. Opened a follow-up PR: #25374